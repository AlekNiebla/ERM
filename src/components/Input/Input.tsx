import * as React from "react";

const Input = () => <span>Input</span>;

export default Input;
