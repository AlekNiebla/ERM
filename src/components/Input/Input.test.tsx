import * as React from "react";
import { shallow } from "enzyme";
// components
import Input from "./Input";

describe("Input Component...", () => {
  it("... renders without errors", () => {
    const wrapper = shallow(<Input />);
    expect(wrapper.find(Input)).toBeTruthy();
  });
  it("... matches snapshot", () => {
    const wrapper = shallow(<Input />);
    expect(wrapper).toMatchSnapshot();
  });
});
