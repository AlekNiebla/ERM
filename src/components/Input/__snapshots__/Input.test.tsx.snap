// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component... ... matches snapshot 1`] = `ShallowWrapper {}`;

exports[`Input Component... should matches snapshot 1`] = `ShallowWrapper {}`;

exports[`Input component... ... matches snapshot 1`] = `ShallowWrapper {}`;
